src/bundle/Resources/public/js/scripts/helpers/timezone.helper.js

```diff
@@ -1,29 +1,61 @@
+        if (timezone) {
+            date = convertDateToTimezone(date, timezone);
+        }
+
         return moment(date).formatICU(format);
     };
-    const formatDateWithTimezone = (date, timezone = userPreferedTimezone, format = userPreferedFullDateFormat) => {
-        return formatDate(convertDateToTimezone(date, timezone), format);
+    const formatFullDateTime = (date, timezone = null, format = userPreferedFullDateTimeFormat) => {
+        return formatDate(date, timezone, format);
     };
-    const formatShortDate = (date, format = userPreferedShortDateFormat) => {
-        return formatDate(date, format);
+    const formatShortDateTime = (date, timezone = null, format = userPreferedShortDateTimeFormat) => {
+        return formatDate(date, timezone, format);
     };
-    const formatShortDateWithTimezone = (date, timezone = userPreferedTimezone, format = userPreferedShortDateFormat) => {
-        return formatDateWithTimezone(date, timezone, format);
+
+    const deprecatedFormatDate = (date, format = userPreferedFullDateTimeFormat) => {
+        console.warn('[DEPRECATED] formatDate function is deprecated');
+        console.warn('[DEPRECATED] it will change behaviour from ezplatform-admin-ui 2.0');
+        console.warn('[DEPRECATED] use formatFullDateTime instead');
+
+        return formatFullDateTime(date, null, format);
+    };
+    const deprecatedFormatShortDate = (date, format = userPreferedShortDateTimeFormat) => {
+        console.warn('[DEPRECATED] formatShortDate function is deprecated');
+        console.warn('[DEPRECATED] it will change behaviour from ezplatform-admin-ui 2.0');
+        console.warn('[DEPRECATED] use formatShortDateTime instead');
+
+        return formatShortDateTime(date, null, format);
+    };
+    const deprecatedFormatDateWithTimezone = (date, timezone = userPreferedTimezone, format = userPreferedFullDateTimeFormat) => {
+        console.warn('[DEPRECATED] formatDateWithTimezone function is deprecated');
+        console.warn('[DEPRECATED] it will be removed from ezplatform-admin-ui 2.0');
+        console.warn('[DEPRECATED] use formatFullDateTime instead');
+
+        return formatFullDateTime(date, timezone, format);
+    };
+    const deprecatedFormatShortDateWithTimezone = (date, timezone = userPreferedTimezone, format = userPreferedShortDateTimeFormat) => {
+        console.warn('[DEPRECATED] formatShortDateWithTimezone function is deprecated');
+        console.warn('[DEPRECATED] it will be removed from ezplatform-admin-ui 2.0');
+        console.warn('[DEPRECATED] use formatShortDateTime instead');
+
+        return formatShortDateTime(date, timezone, format);
     };
 
     eZ.addConfig('helpers.timezone', {
         convertDateToTimezone,
-        formatDate,
-        formatShortDate,
-        formatDateWithTimezone,
-        formatShortDateWithTimezone,
+        formatFullDateTime,
+        formatShortDateTime,
+        formatDate: deprecatedFormatDate,
```

Comment:
What about other bundles which are using those deprecated methods? We should correct them to use not deprecated one

Reply:
there you go 
https://github.com/ezsystems/ezplatform-page-builder/pull/327
https://github.com/ezsystems/ezplatform-admin-ui-modules/pull/173

Reply:
I think there is also one in data-based-publisher.

Reply:
https://github.com/ezsystems/date-based-publisher/pull/97